sys/src/libacpi/harvey.c

```diff
@@ -195,11 +190,9 @@ AcpiOsReadPciConfiguration(ACPI_PCI_ID * PciId,
 }
 
 ACPI_STATUS
-AcpiOsWritePciConfiguration(ACPI_PCI_ID * PciId,
-							UINT32 Reg, UINT64 Value, UINT32 Width)
+AcpiOsWritePciConfiguration(ACPI_PCI_ID * PciId, UINT32 Reg, UINT64 Value, UINT32 Width)
```

Comment:
I like this. I prefer one line and the plan 9 style is not to break on 80 columns. Thanks.